<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWDG</class>
 <widget class="QWidget" name="mainWDG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>239</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Программа слежения и преобразования</string>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QLabel" name="dirNameLBL">
     <property name="text">
      <string>Выбор папки для слежения</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_3">
     <item>
      <widget class="QLineEdit" name="dirNameLED">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dirNamePB">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="logPTE">
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLBL">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Программа предназначена для слежения за директорией и преобразования всех вновь появляющихся в ней файлов.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="restoreACT">
   <property name="text">
    <string>Восстановить</string>
   </property>
  </action>
  <action name="quitACT">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="hideACT">
   <property name="text">
    <string>Спрятать</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quitACT</sender>
   <signal>triggered()</signal>
   <receiver>mainWDG</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restoreACT</sender>
   <signal>triggered()</signal>
   <receiver>mainWDG</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hideACT</sender>
   <signal>triggered()</signal>
   <receiver>mainWDG</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
